roles/openshift_storage_nfs/meta/main.yml

```diff
@@ -11,5 +11,4 @@ galaxy_info:
     - 7
 dependencies:
 - { role: os_firewall }
-- { role: openshift_common }
```

Comment:
If openshift_storage_nfs relies on facts set by openshift_common this will still be needed (or maybe it just needs openshift_facts)?